src/app/routes/connector-ui/contract-agreement-page/contract-agreement-page/contract-agreement-page.service.ts

```diff
@@ -105,4 +123,23 @@ export class ContractAgreementPageService {
       filter((it) => it.isReady),
     );
   }
+
+  private fetchLimits(): Observable<ConnectorLimits | null> {
+    if (this.activeFeatureSet.hasConnectorLimits()) {
+      return this.edcApiService.getEnterpriseEditionConnectorLimits();
+    }
+    return of(null);
+  }
+
+  private setStatus(maxConnectorLimit: number | null) {
```

Comment:
it is NOT a connector limit. it is a consumingContractsLimit, so for example "maxConsumingContracts" (abbreviating ContractAgreements as Contracts)